### ISSUE TITLE
Bug: @update:model-value return _TextLeaf object instead of string

### ISSUE THREAD

**Issue by LavaYasha** [logue/vue-codemirror6#30]
version 1.2.3

```js
<template>
  <div>
    <CodeMirror
      ref="cm"
      :model-value="code"
      @update:model-value="updateCode"
      basic
      dark
      :lang="lang"
    ></CodeMirror>
  </div>
</template>

<script setup lang="ts">
import {onMounted, Ref, ref} from "vue";
import CodeMirror from "vue-codemirror6";
import { LanguageSupport } from '@codemirror/language'
import { sql, PostgreSQL } from "@codemirror/lang-sql";

const props = defineProps<{
  source?: string,
  tables?: { columns: string[] }[],
}>()

const cm = ref()

const code: Ref<string>= ref('')
const updateCode = (val: string) => {
  console.log('value from @update:model-value: ', val)
  code.value = val
}

const lang: LanguageSupport = sql({
  dialect: PostgreSQL,
  schema: {
    tableTest: ['col1', 'col2']
  }
})
</script>


<style scoped lang="scss"></style>
```

![image](https://github.com/logue/vue-codemirror6/assets/56589264/15020a35-dd8c-4ee6-a56b-7305b37d7eec)

**Comment by logue** [comment]
Fixed in 1.2.5.

**Comment by LavaYasha** [comment]
Thanks, this resolved the issue.
